.projenrc.ts

```diff
@@ -131,10 +131,10 @@ const serviceSpecSchemaTask = serviceSpecImporters.addTask('gen-schemas', {
 
 serviceSpecImporters.compileTask.prependSpawn(serviceSpecSchemaTask);
 
-const buildDb = serviceSpecImporters.tasks.addTask('build:db', {
-  exec: 'ts-node src/cli/import-db --force',
+serviceSpecImporters.tasks.addTask('build:db', {
+  exec: 'ts-node src/cli/import-db',
```

Comment:
why no `--force` flag anymore?

Reply:
This projen task used to actually build a proper database as part of the build. Now it's just a helper proxy, so I don't want to set any options.